.depcheckrc.yml

```diff
@@ -15,11 +15,16 @@ ignores:
   - 'rimraf'
   - 'simple-git-hooks'
   - 'ts-node'
-  # Ignore plugins implicitly imported by tools
+  # Ignore plugins for tools
+  - '@typescript-eslint/*'
+  - 'babel-jest'
+  - 'eslint-config-*'
+  - 'eslint-plugin-*'
   - 'jest-silent-reporter'
   - 'prettier-plugin-packagejson'
-  # Ignore plugins we explicitly use with tools
-  - 'babel-jest'
+  - 'typescript-eslint'
+  # Ignore dependencies imported implicitly by tools
+  - 'eslint-import-resolver-typescript'
   # Ignore dependencies which plug into the NPM lifecycle
   - '@lavamoat/preinstall-always-fail'
   - 'react'
```

Comment:
Adding new entries to the ignores section to properly handle ESLint-related dependencies. Matches updates in core https://github.com/MetaMask/core/pull/4727/files#diff-4e83fa6c2fb727ede521c956ea563843a508154426e34a171c6ef4371d6f5959